backend/storj/fs.go

```diff
@@ -22,32 +22,96 @@ import (
 	"storj.io/uplink"
 )
 
+var satMap = map[string]string{
+	"us-central-1.tardigrade.io":  "12EayRS2V1kEsWESU9QMRseFhdxYxKicsiFmxrsLZHeLUtdps3S@us-central-1.tardigrade.io:7777",
+	"europe-west-1.tardigrade.io": "12L9ZFwhzVpuEKMUNUqkaTLGzwY9G24tbiigLiXpmZWKwmcNDDs@europe-west-1.tardigrade.io:7777",
+	"asia-east-1.tardigrade.io":   "121RTSDpyNZVcEU84Ticf2L1ntiuUimbWgfATz21tuvgk3vzoA6@asia-east-1.tardigrade.io:7777",
+}
+
 // Register with Fs
 func init() {
 	fs.Register(&fs.RegInfo{
 		Name:        "storj",
 		Description: "Storj Decentralized Cloud Storage",
 		NewFs:       NewFs,
+		Config: func(name string, configMapper configmap.Mapper) {
+			satelliteString, _ := configMapper.Get("satellite-address")
+			apiKey, _ := configMapper.Get("api-key")
+			passphrase, _ := configMapper.Get("passphrase")
+			accessString, _ := configMapper.Get("access")
+
+			config.FileDeleteKey(name, "satellite-address")
```

Comment:
We should delete these after successfully creating the access and writing it to the config.

Reply:
we will never use it anymore, if we fail or succeed